maps-app/src/main/java/com/esri/android/mapsapp/MapFragment.java

```diff
@@ -605,10 +584,10 @@ private void showSearchBoxLayout() {
 		// Inflating the layout from the xml file
 		mSearchBox = mInflater.inflate(R.layout.searchview, null);
 		// Inflate navigation drawer button on SearchView
-		navButton = (ImageButton) mSearchBox.findViewById(R.id.btn_nav_menu);
+		navButton = mSearchBox.findViewById(R.id.btn_nav_menu);
```

Comment:
All the fields should follow Hungarian Notation rules, so this should be `mNavButton`